### ISSUE TITLE
> just keep running

### ISSUE THREAD

**Issue by CHANGErIBE** [sirius-ai/MobileFaceNet_TF#41]
> just keep running

I keep running for 11 epochs and accuracy in validation set rise to 99.5%,but training accuracy is still always 0. Do you know why? Thank you very much!

_Originally posted by @jolinlinlin in https://github.com/sirius-ai/MobileFaceNet_TF/issues/39#issuecomment-483536646_

**Comment by sirius-ai** [comment]
same with [#39](https://github.com/sirius-ai/MobileFaceNet_TF/issues/39).
